lua/fidget/notification/view.lua

```diff
@@ -86,6 +86,9 @@ end
 ---@param ... string  highlights to apply to text
 ---@return NotificationToken
 local function Token(text, ...)
+  if vim.api.nvim_list_uis()[1].ext_multigrid then
```

Comment:
Thanks for the fix! But I'm not familiar with this API. Is this guaranteed to always return at least one element? And does this ever change while nvim is running?